test/PublishCommand.js

```diff
@@ -2,7 +2,6 @@ import chalk from "chalk";
 import log from "npmlog";
 import normalizeNewline from "normalize-newline";
 import path from "path";
-
```

Comment:
These newlines are significant (and mostly consistent across the other test files), please keep them.

Reply:
Will do